src/components/EditCoursePage/EditCourseForm.test.jsx

```diff
@@ -189,8 +189,8 @@ describe('BaseEditCourseForm', () => {
       id="edit-course-form"
     />);
 
-    const disabledFields = component.find({ disabled: true });
-    expect(disabledFields.length === 1);
+    const disabledFields = component.find({ name: 'mode', disabled: true });
+    expect(disabledFields).toHaveLength(1);
```

Comment:
`expect(disabledFields.length === 1)` will never fail. Had to fix some tests once I noticed they were using this pattern.